sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/implementation/ChatThreadsImpl.java

```diff
@@ -674,7 +674,7 @@ public SendChatMessageResult sendChatMessage(String chatThreadId, SendChatMessag
      * Gets a list of messages from a thread.
      *
      * @param chatThreadId The thread id of the message.
-     * @param maxPageSize The maximum number of messages to be returned per page.
+     * @param maxPageSize The maximum number of messages to be returned per page. The limit can be found from https://docs.microsoft.com/en-us/azure/communication-services/concepts/service-limits.
```

Comment:
You can link to the exact section on the page: https://docs.microsoft.com/en-us/azure/communication-services/concepts/service-limits#size-limits